src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -35,17 +35,15 @@
 #ifndef OPENMS_KERNEL_MSSPECTRUM_H
 #define OPENMS_KERNEL_MSSPECTRUM_H
 
-#include <OpenMS/KERNEL/StandardDeclarations.h>
 #include <OpenMS/METADATA/SpectrumSettings.h>
 #include <OpenMS/METADATA/MetaInfoDescription.h>
 #include <OpenMS/KERNEL/RangeManager.h>
 #include <OpenMS/KERNEL/ComparatorUtils.h>
 #include <OpenMS/METADATA/DataArrays.h>
+#include <OpenMS/KERNEL/StandardTypes.h>
```

Comment:
Thanks pointing out, I corrected the includes changes introduced from cherrypick but obsolete after rebase to recent develop